### PR TITLE
feat: create or import a wallet

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - adds the ability to import a wallet straight from the wallet selection flow.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/4091

None

## Testing

- "Create" routes to native create
- "Import" routes to native import

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

- Develop

image

- This diff

Screenshot 2024-01-23 at 15 42 19

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* `develop`
  * **PR #6061** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6061?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/6061?utm_source=stack-comment).

File: src/context/WalletProvider/SelectModal.tsx [hunk 1]
```diff
@@ -136,24 +142,40 @@ export const SelectModal = () => {
...
         </Grid>
-        <Flex direction={flexDirProp} mt={2} justifyContent='center' alignItems='center'>
-          <Text
-            mb={mbProp}
-            color='text.subtle'
-            translation={walletInfo?.name ? 'common.or' : 'walletProvider.selectModal.footer'}
-          />
-          <Button
-            variant='link'
+        {!walletInfo?.name && (
```
Comment (kaladinlight) [inline src/context/WalletProvider/SelectModal.tsx:145]: don't think we should have this so we have the option available either on no wallet paired, or switch wallet flow (with wallet paired)

Comment (gomesalexandre) [reply src/context/WalletProvider/SelectModal.tsx:145]: The previous flow was for `Don't have a wallet?` to be shown if there's no wallet connected, not sure if it makes sense now.

Probably a question for @reallybeard

Comment (reallybeard) [reply src/context/WalletProvider/SelectModal.tsx:145]: Yah i don't think we need this anymore, should always show I think.

Comment (gomesalexandre) [reply src/context/WalletProvider/SelectModal.tsx]: https://github.com/shapeshift/web/pull/6061/commits/16eb26955f2f0a7966d49816b172050cd852c471

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**